gcovr/tests/test_gcovr.py

```diff
@@ -143,12 +157,23 @@ def run(cmd, cwd=None):
     return returncode == 0
 
 
-def find_reference_files(reference_dir, output_pattern):
-    assert os.path.isdir(reference_dir), "Reference directory exists"
-    for pattern in output_pattern:
-        for reference in glob.glob(os.path.join(reference_dir, pattern)):
-            coverage = os.path.basename(reference)
-            yield coverage, reference
+def find_reference_files(reference_dirs, output_pattern):
+    seen_files = set([])
+    for reference_dir in reference_dirs:
+        for pattern in output_pattern:
+            for dir in REFERENCE_DIRS:
```

Comment:
Looping over both `reference_dirs` and `REFERENCE_DIRS` seems redundant? I think they contain the same values, so that the same files will be selected multiple times. Parts of the following code use `reference_dir`, other parts use `dir`. I think this is incorrect.

Reply:
You're right. I'll remove the inner loop.